boards/arduino-mkr1000/Makefile.include

```diff
@@ -0,0 +1,35 @@
+# define the cpu used by Arduino/Genuino MKR1000 board
+export CPU = samd21
+
+ifeq ($(PROGRAMMER),jlink)
+    # in case J-Link is attached to SWD pins, use a plain CPU memory model
+    export CPU_MODEL = samd21g18a
+
+    # setup JLink for flashing
+    export JLINK_DEVICE := atsamw25
+    include $(RIOTMAKE)/tools/jlink.inc.mk
+
+    # set default serial port depending on operating system
+    PORT_LINUX ?= /dev/ttyUSB0
```

Comment:
This should be ttyACM0 for independent of the programmer, right? So IMHO these two lines should be taken out of this if block.